crates/wasmtime/src/config.rs

```diff
@@ -542,24 +573,6 @@ impl Config {
     pub fn wasm_reference_types(&mut self, enable: bool) -> &mut Self {
         self.features.reference_types = enable;
 
-        #[cfg(compiler)]
-        {
-            self.compiler
-                .set("enable_safepoints", if enable { "true" } else { "false" })
-                .unwrap();
-            // unwind_info must be enabled when either backtraces or reference types are enabled:
-            self.compiler
-                .set(
-                    "unwind_info",
-                    if enable || self.wasm_backtrace {
-                        "true"
-                    } else {
-                        "false"
-                    },
-                )
-                .unwrap();
-        }
-
         // The reference types proposal depends on the bulk memory proposal.
         if enable {
             self.wasm_bulk_memory(true);
```

Comment:
I think that this enable-other-wasm-feature-if-this-is-enabled would be better left to `Engine::new` as well as otherwise this can cause confusing behavior depending on the order of the method calls.